InfluxDb/InfluxDb.php

```diff
@@ -0,0 +1,5 @@
+<?php namespace App\SupportedApps\InfluxDb;
+
+class InfluxDb extends \App\SupportedApps {
```

Comment:
It is also wrong here.
```suggestion
class InfluxDB extends \App\SupportedApps {
```